.website_src/exawind/pazywing.rst

```diff
@@ -1,5 +1,11 @@
 Pazy Wing
 =========
 
+#. `Pazy wing <Pazy_Wing/README.html>`__:  Blade-resolved simulations of a NACA 0018 aeroelastic airfoil in uniform flow to assess pre-flutter static deflection and flutter onset.
 
+   * `Simulation Description and Setup <Pazy_Wing/README.html#simulation-description-and-setup>`__
+   * `Postprocessing <Pazy_Wing/README.html#postprocessing>`__
+   * `Code performance <Pazy_Wing/README.html#code-performance>`__
+   * `Results <Pazy_Wing/README.html#results>`__
 
+`View the case in the github repository <https://github.com/Exawind/exawind-benchmarks/>`__
```

Comment:
We could probably have this point directly to the case, now that the Pazy stuff is merged.

Reply:
Thanks, fixed!